### PR TITLE
'nodejsfetchdone'

### PR DESCRIPTION
**Pull request by capedcrusader23**
Adding feature for nodejs to search ifsc code and display details

### CODE REVIEW

**Comment by capedcrusader23** [comment]
removed the csv file and implemented with ifsc api

**Comment by captn3m0** [comment]
@capedcrusader23 I've made some changes. Your code couldn't have worked because you were trying to return sync from an async method (request.get).

Also added tests. Would be nice if you can go through once and suggest any improvements before I merge.

File: tests/node/client_test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const ifsc = require('./../../src/node/index');
```
Comment (umanghome) [inline tests/node/client_test.js]: Can also use `const ifsc = require('./../../src/node');`

File: src/node/index.js [hunk 2]
```diff
@@ -44,15 +48,29 @@ var lookupNumeric = function(list, code) {
...
 
-var lookupString = function(list, code) {
+let lookupString = function(list, code) {
   return list.indexOf(code) !== -1;
 };
 
-var _fetchDetails = function(code) {
-  throw Error("Not implemented yet");
+let _fetchDetails = function(code, cb) {
+  let url = BASE_URL + code;
```
Comment (umanghome) [inline src/node/index.js]: Make a function `createUrl` and use `createUrl(code)`, would be easier to update the logic in the future if used in multiple places.

**Comment by captn3m0** [comment]
@capedcrusader23 You'll have to wait for the next RBI release (should be sometime this or next week) for this to be released on NPM

Thanks for the contribution :1st_place_medal:

**Comment by capedcrusader23** [comment]
ok, @captn3m0 thanks for correcting my mistakes,leaned efficient way to code.

**Pull request merged**